src/engine/DrawManager.java

```diff
@@ -544,6 +544,37 @@ public void drawNameInput(final Screen screen, final char[] name,
 		}
 	}
 
+	/**
+	 * Draws basic content of game clear screen.
+	 *
+	 * @param screen
+	 *            Screen to draw on.
+	 * @param acceptsInput
+	 *            If the screen accepts input.
+	 * @param isNewRecord
+	 *            If the score is a new high score.
+	 */
+	// CtrlS
+	public void drawGameClear(final Screen screen, final boolean acceptsInput,
+							 final boolean isNewRecord) {
+		String gameClearString = "Game Clear";
```

Comment:
`drawGameClear` 메소드와 `drawGameOver` 메소드가 이 한 줄을 제외하곤 전부 동일한 것 같습니다.

기존 `drawGameOver` 메소드의 이름을 `drawGameEnd` 로 변경하고, `isGameClear` 라는 불리언 변수를 입력 받아 `gameEndString` 을 분기하는 건 어떨까요?

```suggestion
		String gameEndString = isGameClear ? "Game Clear" : "Game Over";
```